voice/bxml/verbs/record.md

```diff
@@ -7,7 +7,7 @@ Bandwidth will keep recordings for up to 30 days. After 30 days the recordings w
 ### Attributes
 | Attribute                    | Description |
 |:-----------------------------|:------------|
-| recordCompleteUrl            | (optional) URL to send the [Record Complete](../callbacks/recordComplete.md) event to once it has ended. Accepts BXML, and may be a relative URL. |
+| recordCompleteUrl            | (optional) URL to send the [Record Complete](../callbacks/recordComplete.md) event to once the recording has ended. Accepts BXML, and may be a relative URL. This callback will not be sent if the recording ended because of a `hangup` event and hence is no longer active. |
```

Comment:
Maybe "...if the recording ended because the call was hung up"?

Reply:
i feel like "hangup event" is more specific and descriptive specifically in the context of our API, no? also, you can have a hang up event without a human being hanging up a phone, which is what i feel like your suggestion is implying.

Reply:
I feel it's incorrect because the hangup event does not end the recording; the recording ended because the call was hung up. Maybe terminated then?

Reply:
okay, good point, let me ponder this